modules/module-mysql/src/common/mysql-to-sqlite.ts

```diff
@@ -107,17 +108,35 @@ export function toSQLiteRow(row: Record<string, any>, columns: Map<string, Colum
       switch (column.typeId) {
         case mysql.Types.DATE:
           // Only parse the date part
-          row[key] = row[key].toISOString().split('T')[0];
+          {
+            const date = row[key] as Date;
+            if (isNaN(date.getTime()) && false) {
```

Comment:
Is the `&& false` intentional/necessary? I might be missing some JS quirk here though.

Reply:
Nope, this was to disable the logic for testing - thanks for catching.

Reply:
Also, there isn't a test for that case yet (a date value of e.g. `2025-00-00`), since the behavior is different between initial replication versus streaming replication (one triggers this condition, the other returns a different date).